src/share/classes/jdk/jfr/EventNames.java

```diff
@@ -0,0 +1,8 @@
+package jdk.jfr;
+
+public class EventNames {
```

Comment:
if this class is part of public API, pls make the Javadoc part more complete.

Reply:
done